src/Compilers/Core/Desktop/CommandLine/CommonCommandLineArguments.cs

```diff
@@ -114,6 +114,11 @@ public abstract class CommandLineArguments
         /// </summary>
         public ImmutableArray<CommandLineSourceFile> AdditionalFiles { get; internal set; }
 
+        /// <value>
+        /// Should the format of error messages include the analyzer that reported the error.
+        /// </value>
+        public bool ReportAnalyzer { get; internal set; }
```

Comment:
I think you mean `<Summary>`, and I would suggest following the style of the summary for `DisplayHelp` and `DisplayLogo`. Maybe the following:

> If true, include the analyzer name in error messages.